content/blog/2022-05-04-e2e-cv-pt2.md

```diff
@@ -0,0 +1,197 @@
+...
+$ dvc exp run --set-param train.batch_size=16
+...
+```
+
+Optionally, you can create delay the execution of the experiments by putting
+them in a
+[queue](https://dvc.org/doc/user-guide/experiment-management/running-experiments#the-experiments-queue),
+and execute them later with the `dvc exp run --run-all` command.
+
+These local experiments are powered by Git references, and you can learn about
+them in [this post](https://dvc.org/blog/experiment-refs). We can display all of
+them with the `dvc exp show` command:
+
+```bash
+$ dvc exp show --only-changed --sort-by=dice_mean
+ ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+  Experiment                Created        train.loss   valid.loss   foreground.acc   jaccard.coeff   dice.multi   dice_mean   acc_mean   train.learning_rate   train.batch_size   models
+ ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+  workspace                 -                 0.10356     0.069076          0.90321         0.75906      0.92371     0.70612    0.97689   0.01                  16                 5854528
+  exp                       Apr 09, 2022      0.13305     0.087599          0.77803         0.66494      0.89084     0.70534    0.97891   0.01                  8                  6c513ae
+  ├── 83a4975 [exp-2d80e]   Apr 09, 2022      0.11189     0.088695          0.86905         0.75296      0.92005     0.70612    0.97689   0.01                  16                 5854528
+  ├── 675efb3 [exp-6c274]   Apr 09, 2022      0.10356     0.069076          0.90321         0.75906      0.92371     0.71492    0.98099   0.1                   16                 770745a
+  └── c8b1857 [exp-04bcd]   Apr 09, 2022      0.11189     0.088695          0.86905         0.75296      0.92005     0.71619    0.98025   0.01                  8                  094c420
+ ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+```
+
+Once we identify one or a few best ones (e.g., highest `dice_mean` score), we
+can either
+[persist](https://dvc.org/doc/user-guide/experiment-management/persisting-experiments)
+them by creating a branch out of an experiment:
+
+```bash
+$ dvc exp branch exp-04bcd my-branch
+Git branch 'my-branch' has been created from experiment 'exp-04bcd'.
+To switch to the new branch run:
+        git checkout my-branch
+```
+
+or we can share them with our teammates by
+[uploading](https://dvc.org/doc/user-guide/experiment-management/sharing-experiments#uploading-experiments)
+them to our git repo:
+
+```bash
+$ dvc exp push origin exp-04bcd
+Pushed experiment 'exp-04bcd'to Git remote 'origin'.
+```
```

Comment:
Not sure if it's worth including `exp push` here given the mention of Studio below. They don't really integrate well at the moment, so it might be more confusing than useful.

Reply:
Related: https://github.com/iterative/studio-support/issues/45

Reply:
removed